core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/hybrid/HybridJDBCConstants.java

```diff
@@ -91,20 +91,20 @@
             "LOWER(UM_USER_NAME)=LOWER(?) AND UM_TENANT_ID=? AND UM_DOMAIN_ID=(SELECT UM_DOMAIN_ID FROM UM_DOMAIN WHERE UM_TENANT_ID=? AND UM_DOMAIN_NAME=?)";
 
     public static final String GET_ROLES =
-            "SELECT UM_ROLE_NAME FROM UM_HYBRID_ROLE WHERE UM_ROLE_NAME LIKE ? AND UM_TENANT_ID=?";
+            "SELECT UM_ROLE_NAME FROM UM_HYBRID_ROLE WHERE UPPER(UM_ROLE_NAME) LIKE ? AND UM_TENANT_ID=?";
```

Comment:
Using the `UPPER` function avoid the index being used in the query execution. This can lead to a performance impact.